recipes/06-laravel-mix.adoc

```diff
@@ -0,0 +1,145 @@
+---
+permalink: laravel-mix
+title: Implementing Laravel Mix
+category: recipes
+---
+= Implementing Laravel Mix
+
+toc::[]
+
+This recipe explains the steps to use link:https://github.com/JeffreyWay/laravel-mix[Laravel Mix] for compiling front-end assets. Laravel Mix provides a clean and easy way to use Webpack, without configuring Webpack with loaders (which can be quiet complex for some users). Do not let the name confuse you, Laravel Mix is not constrained to Laravel. You should have an link:https://adonisjs.com/docs/4.0/installation#_creating_new_app[adonis application created] and a terminal pointed to that directory.
+
+All changes made within a project can also be seen here: link:https://github.com/vincevannoort/adonis-laravel-mix[vincevannoort/adonis-laravel-mix].
+
+== Installing
+Let's start by installing laravel mix in our project as an development dependency.
+[source, bash]
+
+Installing
+----
+npm install laravel-mix  --save-dev
+----
+or
+[source, bash]
+----
+yarn add laravel-mix --dev
+----
+
+Copy the `webpack.mix.js` file
+[source, bash]
+----
+cp -r node_modules/laravel-mix/setup/webpack.mix.js ./
+----
+
+== Setting up our frontend assets
```

Comment:
Can u make this block independent of the other recipe. I don't expect users to read that first and then follow this recipe.